dist/vue-gapi.js

```diff
@@ -5,515 +5,514 @@
  */
 
 (function (global, factory) {
```

Comment:
Feel free to [ignore whitespace](https://github.com/cedpoilly/vue-gapi/pull/74/files?w=1) in here 👇 to see some minor changes caused by the Rollup upgrade.